template.rb

```diff
@@ -51,7 +51,7 @@ def add_solidus_starter_frontend_dependencies
   gem 'canonical-rails'
   gem 'solidus_support'
   gem 'truncate_html'
-  gem 'view_component', require: 'view_component/engine'
+  gem 'view_component', '>= 2.46.0'
```

Comment:
That will take `v3.0`, but that version could bring breaking changes. What about being pessimistic? (`~> 2.46`)